src/components/Pagination/Pagination.stories.tsx

```diff
@@ -1,99 +1,29 @@
 import * as React from "react";
-import { action } from "@storybook/addon-actions";
-import { withDesign } from "storybook-addon-designs";
-import { text } from "@storybook/addon-knobs";
 import { Story } from "@storybook/react/types-6-0";
+import { withDesign } from "storybook-addon-designs";
 
-import Button from "../Button/Button";
-import { ButtonTypes } from "../Button/ButtonTypes";
-import Icon from "../Icons/Icon";
-import { IconNames, IconRotationTypes } from "../Icons/IconTypes";
-import Label from "../Label/Label";
 import Pagination, { PaginationProps } from "./Pagination";
-import Select from "../Select/Select";
 
 export default {
     title: "Pagination",
     component: Pagination,
     decorators: [withDesign],
 };
 
-const selectOptions = (
-    <>
-        <Label htmlFor="paginationSelect" id={"paginationLabel"}>
-            {text("Select Label", "Page ")}
-        </Label>
-        <Select
-            name="Pagination Select"
-            id={"paginationSelect"}
-            isRequired={false}
-            ariaLabel="Pagination Label"
-            labelId={"paginationLabel"}
-            onBlur={action("blur")}
-            onChange={action("changed")}
-            selectedOption={"1 of 7"}
-        >
-            <option aria-selected={true}>
-                {text("Option 1 Text", "1 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 2 Text", "2 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 3 Text", "3 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 4 Text", "4 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 5 Text", "5 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 6 Text", "6 of 7")}
-            </option>
-            <option aria-selected={false}>
-                {text("Option 7 Text", "7 of 7")}
-            </option>
-        </Select>
-    </>
-);
-
-const PaginationTemplate = ({ children, ...args }) => (
-    <Pagination {...args}>{children}</Pagination>
+const PaginationTemplate: Story<PaginationProps> = (args) => (
+    <Pagination {...args}></Pagination>
```

Comment:
Is it a Storybook requirement to pass the props in with spread syntax?  If not, I would prefer to pass the args in explicitly so that anyone looking at this documentation knows what the valid and expected args are!

Reply:
It looks like the other components using a Storybook template are doing it this way. I can investigate if there is another approach.

Reply:
Args now passed in specifically, as requested.